_data/contacts.yaml

```diff
@@ -11,12 +11,13 @@
 - agent: dang
   email: dang.ingress@gmail.com
   area: East Bay
-  form: http://sfbay.resists.org/eby/
+  form: eby/
 
 - agent: dorkus
   email: dorkus@resists.org
-  area: SF
-  form: http://sfbay.resists.org/sfc
+  area: San Francisco
+  form: sfc/signup/
+  hangout: sfc/welcome-hangout/
```

Comment:
Are form and hangout supposed to be on the contacts?  There's nothing to render this information in the index.html file below.